docs/sdk/api/getCredentials.md

```diff
@@ -0,0 +1,46 @@
+# `get-credentials`
+Retrieve commercetools platform credentials from environment variables or file system.
+
+## Install
+
+```bash
+npm install --save @commercetools/get-credentials
+```
+
+## Usage
+
+First `get-credentials` will try to load the credentials from a file `ct-credentials.env` in the current directory and then from the `/etc` directory. If a project's credentials exist in both files the values from the current directory are used. Secondly credentials will be retrieved from an environment variable containing the project key: `CT_<project key>` containing `<client id>:<client secret>`. Notice that a project key will be uppercased and dashes will be replaced by an underscore. So for example: `my-projectkey` becomes `MY_PROJECTKEY`.
+
+Example `ct-credentials.env` file:
+```dosini
+CT_MY_PROJECTKEY=myclientid:mysecret
+CT_OTHER_PROJECTKEY=myclientid:mysecret
+```
+
+## `getCredentials(projectKey)`
+
+Set environment variables from a [dotenv](https://github.com/motdotla/dotenv) file and load the credentials from an environment variable based on the passed project key.
```

Comment:
So the `dotenv` package sets `process.env` variables from the file. But maybe I shouldn't mention it since it's something internal.